src/dal/Jalasoft.TeamUp.Resumes.DAL.Interfaces/IRepository.cs

```diff
@@ -14,5 +15,7 @@ public interface IRepository<T>
         public void Update(int id, T updateObject);
 
         public void Delete(int id);
+
+        public IEnumerable<Resume> GetByName(string name);
```

Comment:
this line should be in IResumeSQLRepository and IResumesInMemoryRepository

Reply:
Done